misk/web/tabs/config/src/components/ConfigComponent.tsx

```diff
@@ -0,0 +1,86 @@
+// tslint:disable-next-line:no-var-requires
+const YAML = require("json2yaml")
+import * as React from "react"
+import styled from "styled-components"
+import { IConfigFile } from "../containers/TabContainer"
+
+interface IConfigProps {
+  files: IConfigFile[]
+  status: string
+}
+
+const Container = styled.div`
```

Comment:
this inline CSS syntax is neat

Reply:
💯 https://www.npmjs.com/package/styled-components